core/core-impl/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptor.java

```diff
@@ -992,9 +1020,11 @@ public long getMaximumValueCount(Solution_ solution) {
     }
 
     /**
+     * @deprecated This was never implemented; any code that depends on this will throw {@link UnsupportedOperationException}.
      * @param solution never null
      * @return {@code >= 0}
      */
+    @Deprecated(forRemoval = true, since = "1.0.0")
     public int getValueCount(Solution_ solution) {
```

Comment:
Normally I'd just remove it, but that would require removing value ratio tabu.
It *doesn't work* and it never worked, throwing an exception.
But technically, it is a backwards incompatible change.